python/example_code/bedrock-runtime/models/anthropic/claude3_chatbot_demo/0_get_models.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import boto3
+
+from utils.custom_logging import setup_custom_logger
+from utils.timeit import timeit
+
+
+class BedrockModelService:
+    """
+    A class to interact with the Bedrock AI service, specifically for operations
```

Comment:
Change 'Bedrock' to 'Amazon Bedrock' (first use in file)

Reply:
Done